voice/bxml/callbacks/gather.md

```diff
@@ -25,11 +25,14 @@ Content-Type: application/xml; charset=utf-8
 | from             | The phone number that made the call, in E.164 format (e.g. +15555555555).                                         |
 | direction        | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes.                 |
 | callId           | The call id associated with the event.                                                                            |
+| parentCallId     | (optional) The call id of the original call leg that contained the `<Transfer>` tag. Only set on B-leg callbacks.|
```

Comment:
For all of these things that use this phrasing, it might read better to lead with the reason it might be there. Maybe something like "If this event is related to the B leg of a transfer, the call ID of the original call leg that contained the <Transfer>, otherwise null"?